packages/legend-application-studio/src/components/editor/side-bar/Explorer.tsx

```diff
@@ -291,6 +295,55 @@ const ExplorerContextMenu = observer(
       (type: string): (() => void) =>
       (): void =>
         editorStore.newElementState.openModal(type, _package);
+    const isDependencyProjectRoot = (): boolean =>
```

Comment:
Can't we just compare the node.package to see if its one of the packages of dependencyManager ? Don't like really comparing labels as in theory we can always change the label